source/1_engine/rex_engine/include/rex_engine/memory/win/win_mem_stats.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
Do we want these conditional compilation statements or do we exclude the file in general from the build?